### ISSUE TITLE
Failed to load resource: net::ERR_NETWORK_CHANGED asyncToGenerator.js - Transaction is still reported as in the mempool 

### ISSUE THREAD

**Issue by timstackblock** [blockstack/stacks-blockchain-sidecar#144]
**Describe the bug**
This is the error that was posted in the slack chat on Friday June 12
https://blockstackpbc.slack.com/archives/CHY9G2PP1/p1591966531230200

```
1 Unchecked runtime.lastError: The message port closed before a response was received.
sidecar.staging.blockstack.xyz/v2/accounts/STK0RK9Z7RV289CT6JVBV79HN4J34WNGWMZDZ4VY:1 Failed to load resource: net::ERR_NETWORK_CHANGED
asyncToGenerator.js:6 Uncaught (in promise) TypeError: Failed to fetch
```

> COMMENT FROM MATT

Shortly after testnet restart, we tested a restart of the sidecar by itself. from the logs, it halted from detecting multiple mempool tx updates with same txid. @zone117x  will change this from an error to a log warning if/when it happens again.

STX ADDRESS: STK0RK9Z7RV289CT6JVBV79HN4J34WNGWMZDZ4VY

TXID 0x35eaf218ad1561a15a6055aeb13d28ba3e965fe23bc6657201c764cb0fb67f21

https://testnet-explorer.blockstack.org/txid/0x35eaf218ad1561a15a6055aeb13d28ba3e965fe23bc6657201c764cb0fb67f21

https://sidecar.staging.blockstack.xyz/sidecar/v1/tx/0x35eaf218ad1561a15a6055aeb13d28ba3e965fe23bc6657201c764cb0fb67f21

**To Reproduce**
1. Go to the test demo page and send stx using the faucet
2. Click on the explorer transaction link and you will see an error in explorer
3. View the console error and you will see a sidecar error 
4. See error messaging below

**Expected behavior**
Expected behavior is for this transaction to go through without issue and display a successful transaction page in explorer

**Screenshots**
MESSAGE THE USERS SEES WHEN THE ERROR OCCURS
<img width="791" alt="Screen Shot 2020-06-12 at 9 04 23 AM" src="https://user-images.githubusercontent.com/49165468/84784519-687bab80-afb8-11ea-883b-0089fdc7c943.png">

WHEN THIS HAPPENS THE EXPLORER WILL READ PENDING AND THERE WILL BE AN ERROR MESSAGE THAT STATES THE TRANSACTION WAS ROLLED BACK
<img width="1405" alt="Screen Shot 2020-06-12 at 8 49 46 AM" src="https://user-images.githubusercontent.com/49165468/84784538-6f0a2300-afb8-11ea-8966-70865da10047.png">

**Console log**
CONSOLE LOG ERROR BELOW SHOWS THE SIDECAR ERROR
<img width="1437" alt="Screen Shot 2020-06-12 at 9 05 51 AM" src="https://user-images.githubusercontent.com/49165468/84784261-1cc90200-afb8-11ea-9b28-2a2fc2952e10.png">

**Desktop (please complete the following information):**
 - OS: iOS 10.14.4
 - Browser Chrome

**Comment by zone117x** [comment]
Looks like a few different issues here:
1. Duplicate mempool txs stalled the sidecar, and sending txs was non-functional during this time. This was fixed and deployed a couple days ago with https://github.com/blockstack/stacks-blockchain-sidecar/pull/142.
2. The explorer showing the rollback error for pending transactions is just a display issue and should be fixed once https://github.com/blockstack/explorer/pull/190 is merged.
3. `sidecar.staging.blockstack.xyz/v2/accounts/STK0RK9Z7RV289CT6JVBV79HN4J34WNGWMZDZ4VY:1 Failed to load resource: net::ERR_NETWORK_CHANGED` -- this is a proxied request to the core-node RPC server. I'm assuming this happened because the system was stalled. We would need this reproducible otherwise I don't think it's an issue.


@timstackblock is any of this reproducible right now? If not I think we can close. 

I tested out the STX faucet at https://authenticator-demo.netlify.app/ and the tx went through just fine, even though there's a front-end error `Something went wrong when requesting the faucet` because of 

![image](https://user-images.githubusercontent.com/1447546/84787732-a267a480-afee-11ea-8136-b06870901035.png)
